vllm/model_executor/models/gpt_bigcode.py

```diff
@@ -299,4 +299,10 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
             param = params_dict[name]
             weight_loader = getattr(param, "weight_loader",
                                     default_weight_loader)
-            weight_loader(param, loaded_weight)
+
```

Comment:
Can you just add a comment with:

`TODO (@robertgshaw2-neuralmagic): refactor`

Reply:
Done.